app/decorators/controllers/hyrax/downloads_controller_decorator.rb

```diff
@@ -0,0 +1,30 @@
+module Hyrax
+  DownloadsController.class_eval do
+    # `asset` is inherited from hydra-head/hydra-core/app/controllers/concerns/hydra/controller/download_behavior.rb
+    def send_content
+      super
+      update_download_stats(asset)
+    end
+
+    def download_stats(current_file)
```

Comment:
Because these methods have nothing to do with HTTP interaction they could move out of the controller to a service class, so that the only addition on line 6 needs to be: 
```
DownloadStats.update(asset)
```